src/main/kotlin/no/nb/bikube/newspaper/service/CollectionsService.kt

```diff
@@ -232,4 +259,9 @@ class CollectionsService  (
             }
     }
 
+    private fun filterByFormat(itemPartReference: CollectionsPartsReference?, isDigital: Boolean) =
+        itemPartReference?.getFormat() == if (isDigital) CollectionsFormat.DIGITAL else CollectionsFormat.PHYSICAL
+
+    private fun filterByYear(manifestationPartsObject: CollectionsPartsObject, date: LocalDate) =
+        manifestationPartsObject.partsReference?.dateStart?.first()?.dateFrom == date.toString()
```

Comment:
This could be moved to an extension function as well, `CollectionsPartsObject.getDateStr()` for example

Reply:
There is an extension function for dates from parts (CollectionsPartsReference.getDate())

Reply:
Should probably also rename the function as it filters by full date, not year